src/db/results.rs

```diff
@@ -21,7 +21,21 @@ pub type DeleteBsos = SyncTimestamp;
 pub type DeleteBso = SyncTimestamp;
 pub type PutBso = SyncTimestamp;
 
-pub type CreateBatch = String;
+#[derive(Debug, Clone)]
```

Comment:
nit: the defaults for both CreateBatch/GetQuotaUsage are their basic inner type's defaults, so you don't even have to implement it yourself (let derive Default do it)

```suggestion
#[derive(Debug, Default, Clone)]
```